students/junghyun/users/urls.py

```diff
@@ -0,0 +1,8 @@
+
+from django.urls import path
+
+from users.views  import UsersView
+
+urlpatterns = [
+    path('', UsersView.as_view()),
+]
```

Comment:
이 부분도 아래 `View Class`의 이름을 수정하면서 같이 수정해주시면 되겠습니다!
그리고 라운지에서 말씀드린 내용대로 회원가입 기능에 해당하는 `SignUpView`를 맵핑해주는 경우, 그에 맞게 주소도 작성해주세요!
```suggestion
from users.views  import SignUpView

urlpatterns = [
    path('/signup', SignUpView.as_view()),
]
```

Reply:
수정사항이 하나도 반영이 안된 상태로 푸쉬가 되었네요 ㅠㅠ 다시 했습니다!